govc/USAGE.md

```diff
@@ -5040,6 +5048,123 @@ Options:
   -vapp.ipath=           Find vapp by inventory path
 ```
 
+## vcsa.access.consolecli.get
+
+```
+Usage: govc vcsa.access.consolecli.get [OPTIONS]
+
+Get enabled state of the console-based controlled CLI (TTY1).
+Examples:
+govc vcsa.access.consolecli.get
+
+Options:
+```
+
+## vcsa.access.consolecli.set
+
+```
+Usage: govc vcsa.access.consolecli.set [OPTIONS]
+
+Set enabled state of the console-based controlled CLI (TTY1).
+Examples:
+# Enable Console CLI
+govc vcsa.access.consolecli.set -consolecli-enabled=True
```

Comment:
nit: all these `enabled` fields could be shortened and simplified to `--enabled`

Reply:
updated to `enabled`